packages/cli/src/PublicApi/v1/handlers/executions/executions.service.ts

```diff
@@ -110,6 +110,6 @@ export async function getExecutionInWorkflows(
 	});
 }
 
-export async function deleteExecution(execution: IExecutionBase): Promise<DeleteResult> {
-	return Container.get(ExecutionRepository).deleteExecution(execution.id as string);
+export async function deleteExecution(execution: IExecutionBase): Promise<void> {
+	return Container.get(ExecutionRepository).softDeleteExecution(execution.id as string);
```

Comment:
Just as a reminder, we'll need to filter out soft-deleted executions from all the executions we're serving to the FE.

Reply:
yeah. I still need to update some of the executions queries to explicitly exclude soft-deleted rows. That's probably the biggest reason this PR is still in draft.